Gyro not building with zig `0.10.0`

Gyro doesn't build currently on macOS using zig `0.10.0` even though it's listed as the supported version on the readme. 

```
zig-macos-aarch64-0.10.0/lib/std/build.zig:2314:31: error: deprecated; use addIncludePath
    pub const addIncludeDir = @compileError("deprecated; use addIncludePath");
                              ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

--- a/.github/workflows/linux.yml
+++ b/.github/workflows/linux.yml
@@ -4,8 +4,6 @@ on:
     branches: [ master ]
   pull_request:
     branches: [ master ]
-  schedule:
-    - cron: "0 7 * * *"
 
 jobs:
   build:
@@ -21,7 +19,7 @@ jobs:
     - name: Setup Zig
       uses: goto-bus-stop/setup-zig@v1.3.0
       with:
-        version: 0.10.0
+        version: 0.10.1
 
     - name: Build 32-bit
       run: zig build test -Dtarget=i386-linux-musl --prefix tmp

--- a/.github/workflows/macos.yml
+++ b/.github/workflows/macos.yml
@@ -4,8 +4,6 @@ on:
     branches: [ master, gyro ]
   pull_request:
     branches: [ master ]
-  schedule:
-    - cron: "0 7 * * *"
 
 jobs:
   build:
@@ -21,7 +19,7 @@ jobs:
     - name: Setup Zig
       uses: goto-bus-stop/setup-zig@v1
       with:
-        version: 0.10.0
+        version: 0.10.1
 
     - run: echo "$(pwd)/zig-out/bin" >> $GITHUB_PATH
 

--- a/.github/workflows/windows.yml
+++ b/.github/workflows/windows.yml
@@ -4,8 +4,6 @@ on:
     branches: [ master ]
   pull_request:
     branches: [ master ]
-  schedule:
-    - cron: "0 7 * * *"
 
 jobs:
   build:
@@ -29,7 +27,7 @@ jobs:
     - name: Setup Zig
       uses: goto-bus-stop/setup-zig@v1
       with:
-        version: 0.10.0
+        version: 0.10.1
 
     - name: Build 32-bit
       run: zig build test -Dtarget=i386-windows --prefix tmp

--- a/README.md
+++ b/README.md
@@ -76,7 +76,7 @@ a branch, tag, or commit hash, if not specified, gyro uses the default branch.
 In order to install gyro, all you need to do is extract one of the [release
 tarballs](https://github.com/mattnite/gyro/releases) for your system and add the
 single static binary to your PATH. Gyro requires the latest stable version of
-the Zig compiler (0.10.0 at time of writing)
+the Zig compiler (0.10.1 at time of writing)
 
 ### Building
 

